### PR TITLE
Updating info for postgreqsql metrics

### PR DESCRIPTION
**Pull request by shyamvalsan**
Updating info for: 
- db_buffer_cache_hit_ratio
- db_rows_read_ratio
- db_rows_read
- db_conflicts
- db_conflicts_stat

##### Test Plan

Tested on local agent dashboard

### CODE REVIEW

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -3951,19 +3951,19 @@ netdataDashboard.context = {
             ],
         },
         */        
-        info: 'Percentage of returned/fetched rows.'
+        info: 'Percentage of fetched vs returned rows. This metric measures the percentage of rows that contain data needed to execute the query, out of the total number of rows scanned. A high value indicates that the database is executing queries efficiently, while a low value indicates that the database is performing extra work by scanning a large number of rows that aren\'t required to process the query. Low values may be caused by missing indexes or inefficient queries.'
```
Comment (ilyam8) [inline web/gui/dashboard_info.js:3954]:
```suggestion
        info: 'The percentage of rows that contain data needed to execute the query, out of the total number of rows scanned. A high value indicates that the database is executing queries efficiently, while a low value indicates that the database is performing extra work by scanning a large number of rows that aren\'t required to process the query. Low values may be caused by missing indexes or inefficient queries.'
```

How about this? I think if there is a way to shorten the text (without losing information), we should always do so.

**Review by ilyam8 — DISMISSED** [review]

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -3951,19 +3951,19 @@ netdataDashboard.context = {
...
+        info: '<p>Read queries throughput.</p><p><b>Returned</b> - Total number of rows scanned by queries. This value indicates rows returned by the storage layer to be scanned, not rows returned to the client. <b>Fetched</b> - Subset of scanned rows (<b>Returned</b>) that contained data needed to execute the query.</p>'
     },
     'postgres.db_rows_written': {
         info: '<p>Write queries throughput.</p><p><b>Inserted</b> - number of rows inserted by queries. <b>Deleted</b> - number of rows deleted by queries. <b>Updated</b> - number of rows updated by queries.</p>'
     },
     'postgres.db_conflicts': {
-        info: 'Number of queries canceled due to conflicts with recovery. Conflicts occur only on standby servers.'
+        info: 'Number of queries canceled due to conflict with recovery on standby servers. To minimize query cancels caused by cleanup records consider configuring <a href="https://www.postgresql.org/docs/9.4/runtime-config-replication.html#GUC-HOT-STANDBY-FEEDBACK" target="_blank"><i>hot_standby_feedback</i></a>.'
```
Comment (thiagoftsm) [inline web/gui/dashboard_info.js:3963]: Is there a special reason to use a link for `PostgreSQL 9.4` instead tne [current](https://www.postgresql.org/docs/current/runtime-config-replication.html#RUNTIME-CONFIG-REPLICATION-STANDBY)?

**Review by thiagoftsm — DISMISSED** [review]
My doubt is considered a minor improvement, so I am approving the PR.

File: web/gui/dashboard_info.js (hunk 2 above)
Comment (ilyam8) [inline web/gui/dashboard_info.js:3963]:
```suggestion
        info: 'Number of queries canceled due to conflict with recovery on standby servers. To minimize query cancels caused by cleanup records consider configuring <a href="https://www.postgresql.org/docs/current/runtime-config-replication.html#GUC-HOT-STANDBY-FEEDBACK" target="_blank"><i>hot_standby_feedback</i></a>.'
```

Nice catch, @thiagoftsm

**Review by ilyam8 — APPROVED** [review]

File: web/gui/dashboard_info.js (hunk 2 above)
Comment (shyamvalsan) [reply web/gui/dashboard_info.js]: Good catch.

Comment (shyamvalsan) [reply web/gui/dashboard_info.js]: Thanks - good catch.

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**